Compile error

- __SerialPort Version__: 6.1.0
- __NodeJS Version__: 9.5.0
- __Operating System__ and __Hardware Platform__: Alpine Linux Docker

### Summary of Problem

  CXX(target) Release/obj.target/serialport/src/serialport_linux.o
../src/serialport_linux.cpp: In function 'int linuxSetCustomBaudRate(int, unsigned int)':
../src/serialport_linux.cpp:10:18: error: 'TCGETS2' was not declared in this scope
     if(ioctl(fd, TCGETS2, &t)) {
                  ^~~~~~~
../src/serialport_linux.cpp:18:18: error: 'TCSETS2' was not declared in this scope
     if(ioctl(fd, TCSETS2, &t)) {
                  ^~~~~~~
../src/serialport_linux.cpp: In function 'int linuxGetSystemBaudRate(int, int*)':
../src/serialport_linux.cpp:29:16: error: 'TCGETS2' was not declared in this scope
   if(ioctl(fd, TCGETS2, &t)) {
                ^~~~~~~
make: *** [serialport.target.mk:98: Release/obj.target/serialport/src/serialport_linux.o] Error 1

### Steps and Code to Reproduce the Issue
npm install

--- a/src/serialport_linux.cpp
+++ b/src/serialport_linux.cpp
@@ -1,6 +1,7 @@
 #if defined(__linux__)
 
 #include <sys/ioctl.h>
+#include <asm/ioctls.h>
 #include <asm/termbits.h>
 
 // Uses the termios2 interface to set nonstandard baud rates

